app.json

```diff
@@ -1,24 +1,24 @@
 {
-    "id": "0d3ac5b3-dd0b-43d3-924a-5a7433902589",
-    "version": "1.0.0",
-    "requiredApiVersion": "^1.17.0",
-    "iconFile": "icon.png",
-    "author": {
-        "name": "Rocket.Chat",
-        "homepage": "https://github.com/RocketChat/Apps.Salesforce.LiveAgent",
-        "support": "https://github.com/RocketChat/Apps.Salesforce.LiveAgent/issues"
-    },
-    "name": "Saleforce Live Agent Integration",
-    "nameSlug": "salesforce-live-agent-integration",
-    "classFile": "SalesforceLiveAgentApp.ts",
-    "description": "Integration between Rocket.Chat and the Salesforce Live Agent (Chat). Please refer our Github repository for setup instructions.",
-    "implements": [
-        "IPostMessageSent",
-        "IPostLivechatAgentAssigned",
-        "IPostLivechatAgentUnassigned",
-        "IPostLivechatRoomClosed",
-        "IRoomUserTyping",
-        "IUIKitLivechatInteractionHandler"
-    ],
-    "commitHash": "29fb3e0d0ff0e996e4770fd1d7a44f4cfb65d938"
+	"id": "0d3ac5b3-dd0b-43d3-924a-5a7433902589",
+	"version": "1.0.0",
+	"requiredApiVersion": "WideChat/Rocket.Chat.Apps-engine.git#9a89addfacd4600f4e49d8dc7ce96a0ef88669ea",
```

Comment:
good catch!  Assuming this passes.....  let's just remember to revert before we eventually merge.  Thanks!